src/pages/en/reference/api-reference.md

```diff
@@ -274,7 +274,7 @@ export async function getStaticPaths() {
   return [
     { params: { id: '1' } },
     { params: { id: '2' } },
-    { params: { id:  3 } }
+    { params: { id:  3 // Can be a number too! Only strings and numbers are supported values. } }
```

Comment:
```suggestion
    { params: { id:  3  }  }  // Can be a number too! Only strings and numbers are supported values.
```
I'm going to ask @delucis to confirm, but I think we would typically only comment after the complete line of code, not within it. That would be my instinct.

Also, I could go either way re: repeating "Only strings and numbers are supported" in the code comment. I do like that it's clear! (Especially for those who only skim and read the code samples. 😅 ) 

I suspect that just "Can be a number too!" would also be fine, given we have just said that same thing above. I think the key thing to point out is that no, we didn't make a typo! We meant for that to be a number.  ;)